applications/api-cat/src/main/java/no/acat/harvester/ApiHarvester.java

```diff
@@ -76,8 +76,12 @@ void indexApi(ApiDocument document) throws JsonProcessingException {
         logger.info("ApiDocument is indexed. id={}, url={}", document.getId(), document.getApiSpecUrl());
     }
 
-    List<ApiCatalogRecord> getApiCatalog() {
-        List<ApiCatalogRecord> result = new ArrayList<>();
+    List<ApiRegistrationPublic> getApiRegistrations() {
+        return getApiRegistrationsFromCsv();
+    }
+
+    List<ApiRegistrationPublic> getApiRegistrationsFromCsv() {
+        List<ApiRegistrationPublic> result = new ArrayList<>();
 
         org.springframework.core.io.Resource apiCatalogCsvFile = new ClassPathResource("apis.csv");
```

Comment:
Er dette en mock / hardcode for development ? (altså at vi laster inn APIRegistrations fra en fil)